### ISSUE TITLE
[Enhancement] Make GUI elements bigger or DPI aware

### ISSUE THREAD

**Issue by baack** [ibbaa/keepitup#7]
[Enhancement] Make GUI elements bigger or DPI aware.

Currently all the buttons are quite small, almost as for stylus, not a finger.

**Comment by ibbaa** [comment]
All GUI elements are already DPI aware (or at least should be  :) There are different sized icons for different resolutions and all elements are scaled by providing dp sizes. They can be to small anyway :-) It's difficult to test, since I don't have so much devices (on my main smartphone it's fine for me, maybe little small) and also the emulator devices are limited, especially when it comes to screen sizes. The icons are painted by hand with gimp and it was absolutely pain and I won't do that again. But I can have a look at the scaling.

**Comment by baack** [comment]
> The icons are painted by hand with gimp and it was absolutely pain and I won't do that again. But I can have a look at the scaling.

Many icons are available for free (as in freedom), with or without author mentioning. I would advice to use one of them to avoid developer drawing in Gimp or Inkscape.

Crystal Project can be sufficient: https://commons.wikimedia.org/wiki/Category:Crystal_Project

**Comment by ibbaa** [comment]
You are right. Don't know what drove me to do it by myself. But replacing all icons is also some effort. It's lower priority than the pings.

**Comment by ibbaa** [comment]
Won't change the icons in the foresseable future. The current icons are good enough.
